ios/Plugin/CapacitorUpdater.swift

```diff
@@ -555,75 +529,183 @@ extension CustomError: LocalizedError {
         UserDefaults.standard.synchronize()
         print("\(self.TAG) Current bundle set to: \((bundle ).isEmpty ? BundleInfo.ID_BUILTIN : bundle)")
     }
+    private func getChecksum(filePath: URL) -> String {
```

Comment:
I believe that this function was renamed: https://github.com/Cap-go/capacitor-updater/commit/c309a6f58f706d997ad33f44dba8aab4cad2eb36